lib/utils/package-manager.spec.js

```diff
@@ -65,7 +72,7 @@ describe("package-manager", () => {
 	it("should spawn npm install from spawnChild", () => {
 		const packageName = "some-pkg";
 
-		mockSpawnErrorOnce();
+		mockSpawnErrorTwice();
```

Comment:
May I ask, what the reason for having same spawn calls twice instead of one ?

Reply:
`spawnChild` now calls `spawn.sync` twice, because it calls `getPackageManager` twice. It calls `getPackageManager` once and also calls `getPathToGlobalPackages`. which in turn calls `getPackageManager`. I can refactor it if that seems too redundant.